keyboards/frosty_flake/readme.md

```diff
@@ -1,16 +1,42 @@
-Frosty Flake Controller
-===
+# Frosty Flake Controller
 
-This is the firmware for Rev. 20140521 of the Frosty Flake controller by [Bathroom Epiphanies](http://bathroomepiphanies.com/controllers/), a replacement controller for the [Cooler Master Quick Fire Rapid](http://www.coolermaster.com/peripheral/keyboards/quickfirerapid/).
+This is the firmware for Rev. 20140521 of the Frosty Flake controller
+by [Bathroom Epiphanies](http://bathroomepiphanies.com/controllers/),
+a replacement controller for the [Cooler Master Quick Fire
+Rapid](http://www.coolermaster.com/peripheral/keyboards/quickfirerapid/).
 
-The code was adapted from the [BathroomEpiphanies TMK Firmware](https://github.com/BathroomEpiphanies/epiphanies_tmk_keyboard/tree/master/be_controllers), but has been cleaned up to match the [schematic](https://deskthority.net/wiki/File:Frosty_Flake_Schematics.pdf) and gone through some minor refactoring for QMK.
+The code was adapted from the [BathroomEpiphanies TMK
+Firmware](https://github.com/BathroomEpiphanies/epiphanies_tmk_keyboard/tree/master/be_controllers),
+but has been cleaned up to match the
+[schematic](https://deskthority.net/wiki/File:Frosty_Flake_Schematics.pdf)
+and gone through some minor refactoring for QMK.
 
-Keyboard Maintainer: QMK Community
-Hardware Supported: Frosty Flake
-Hardware Availability: https://1upkeyboards.com/qfr-frosty-flake-controller.html
+## 104 and 87 layout support
+
+Support for both 104 key and 87 key layouts is provided. See the
+keymaps `default` (104) and `tkl` (87) for example layouts.
+
+**Keyboard Maintainer:** QMK Community
+
+**Hardware Supported:** Frosty Flake
+
+**Hardware Availability:** https://1upkeyboards.com/qfr-frosty-flake-controller.html
```

Comment:
Our readme standard puts these on 3 consecutive lines without the bolding. You should put 2 spaces at the end so the line break makes it to the html. Could you please update this to comply with that?